src/phantom/schema.py

```diff
@@ -22,7 +25,25 @@ class Schema(TypedDict, total=False):
     maxLength: Optional[int]
 
 
+def desc_from_docstring(cls) -> str:
+    # ds = (next(filter(lambda x: x.__doc__, cls.__mro__), cls).__doc__ or "").strip()
+    ds = (cls.__doc__ or "").strip()
+
+    if not ds:
+        return ""
+    lines = ds.split("\n", maxsplit=1)
+    if len(lines) == 1:
+        return ds
+    rest = textwrap.dedent(lines[1])
+    if fst := lines[0].strip():
+        return f"{fst}\n{rest}"
+    else:
+        return rest
+
+
 class SchemaField:
+    __use_docstring__: ClassVar[str]
```

Comment:
issue: I think this should be `bool`, not `str`?

Reply:
as it stands you're right, thanks. Might be I have to reconsider this depending on how I implement your desired behavior for case D.